pkg/private/pkg_files.bzl

```diff
@@ -259,6 +259,10 @@ def add_label_list(
       default_group: fallback mode to use for Package*Info elements without group
     """
 
+    if hasattr(ctx.attr, "include_runfiles"):
+        include_runfiles = ctx.attr.include_runfiles
+    else:
+        include_runfiles = False
```

Comment:
This could be shortened to

```starlark
include_runfiles = getattr(ctx, "include_runfiles", False)
```

Reply:
I changed my mind.   That change actually gives a different result. The getattr with keeps yielding False, even when set.